strophejingle/index.html

```diff
@@ -14,7 +14,7 @@
       <form id="loginInfo">
         <label>JID: <input id="jid" type="text" name="jid" placeholder="me@example.com"/></label>
         <label>Password: <input id="password" type="password" name="password" placeholder="secret"/></label>
-        <label>BOSH URL: <input id="boshURL" type="text" name="boshURL" placeholder="/http-bind"/></label>
+        <label>BOSH URL: <input id="boshURL" type="text" name="boshURL" value="/http-bind/"/></label>
```

Comment:
Yes. That is true. I was really tired of filling that field over and over again at some point, and i guess i forgot it there.

Thanks